internal/pool/pool.go

```diff
@@ -78,7 +78,8 @@ type ConnPool struct {
 
 	stats Stats
 
-	_closed uint32 // atomic
+	_closed   uint32 // atomic
+	_closedCh chan struct{}
```

Comment:
Please remove leading `_`. `_closed` has it because there is a method called `closed`.